src/utils/url.ts

```diff
@@ -39,7 +40,15 @@ export const openRoomUrl = (url: string) => {
 };
 
 export const openUrl = (url: string) => {
-  window.open(url, "_blank", "noopener,noreferrer");
+  isValidUrl(url)
+    ? window.open(url, "_blank", "noopener,noreferrer")
+    : console.error(
+        `Invalid URL ${url} on page ${window.location.href}; ignoring`
+      );
```

Comment:
@thecadams this is a perfect place where we can make use of the `Bugsnag.notify` helper to explicitly report an error, docs for usage:

- https://docs.bugsnag.com/platforms/javascript/#reporting-handled-errors

Reply:
Will do a short followup PR to fix this + any other console.error's that we would benefit from gaining insight into

Reply:
https://github.com/sparkletown/sparkle/pull/889